cfy_manager/main.py

```diff
@@ -806,7 +799,10 @@ def remove(verbose=False, force=False):
     """ Uninstall Cloudify Manager """
 
     _prepare_execution(verbose)
-    _validate_force(force, 'remove')
+    if force:
+        logger.warning('--force is deprecated, does nothing, and will be '
```

Comment:
I think we can lose the does nothing part, the deprecated is clear ee

Reply:
eh i like to be explicit
because otherwise people could ask - "ok, this is deprecated, so what do i use instead?". And so now it is clear that since this does nothing, you use nothing instead